### PR TITLE
Expose a `fused_topk_raw_logits` API

### PR DESCRIPTION
**Pull request by zianglih**
## 📌 Description
Expose a raw logits topk api for existing code in trtllm fused moe.
Currently the implementation is not optimized and perf is wrose than a naive torch implementation here https://github.com/sgl-project/sglang/pull/19537/commits/6ed9c53b92657380aed07f822ac9af2ae6a60764

## 🔍 Related Issues

https://github.com/flashinfer-ai/flashinfer/issues/2676
https://github.com/sgl-project/sglang/pull/19537
## 🚀 Pull Request Checklist

Thank you for contributing to FlashInfer! Before we review your pull request, please make sure the following items are complete.

### ✅ Pre-commit Checks

- [ ] I have installed `pre-commit` by running `pip install pre-commit` (or used your preferred method).
- [ ] I have installed the hooks with `pre-commit install`.
- [ ] I have run the hooks manually with `pre-commit run --all-files` and fixed any reported issues.

> If you are unsure about how to set up `pre-commit`, see [the pre-commit documentation](https://pre-commit.com/).

## 🧪 Tests

- [ ] Tests have been added or updated as needed.
- [ ] All tests are passing (`unittest`, etc.).

### CODE REVIEW

File: csrc/moe_utils_binding.cu [hunk 1]
```diff
@@ -345,3 +344,63 @@ void moe_sort(
...
+
+#ifdef ENABLE_BF16
+// ============================ fused_topk_raw_logits bindings ============================
+// Use TRTLLM routingRenormalize code path (same kernel family as non-routed TRTLLM MoE).
+//
+// Notes:
+// - This function computes top-k scores/weights from raw routing logits (mPtrScores path).
+// - mPtrTopKIds is intentionally left nullptr to force score-driven routing selection.
+// - topk_packed_ptr is required by routing kernels for large-token paths.
+void fused_topk_raw_logits_trtllm_renormalize(
```
Comment (yzh119) [inline csrc/moe_utils_binding.cu:357]: I have some concern about the API name, can we make it simpler? (and why do we call it `fused`?)
